packages/governance/test/swingsetTests/contractGovernor/governedContract.js

```diff
@@ -0,0 +1,49 @@
+// @ts-check
+
+import { Far } from '@agoric/marshal';
+import { E } from '@agoric/eventual-send';
+import { assert, details as X } from '@agoric/assert';
+import { sameStructure } from '@agoric/same-structure';
+
+import { buildParamManager, ParamType } from '../../../src/paramManager';
+
+const MALLEABLE_NUMBER = 'MalleableNumber';
+
+export const governedParameterTerms = {
+  contractParams: [MALLEABLE_NUMBER],
+};
+
+/* @type {ContractStartFn} */
```

Comment:
```suggestion
/** @type {ContractStartFn} */
```

Without the two stars at the start, it doesn't type.

Reply:
fixed